### DISCUSSION TITLE
Mark some non-stroke paths as strokes

### DISCUSSION THREAD

**Discussion by Blipz** [Azgaar/Armoria#167] [Ideas]
I think it would be nice to declare as "pseudo-strokes" (with a class, similarly as secondary/tertiary) in the charge SVG the paths that the user expects to be strokes but actually are not, so that they could be colored as if they were.
It would be a bit tedious to check all charges, but doable (maybe most of it can be automated).

**Comment by Azgaar** [comment]
Yes, the charges are messy in terms of strokes. They should work the same even if technically there are not. Good idea.

**Comment by Blipz** [comment]
Done here: https://github.com/Azgaar/Armoria/pull/171
